src/Tasks/Common/Resources/Strings.resx

```diff
@@ -758,4 +758,12 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1163: Cannot emit symbols when publishing for .NET 5 with Crossgen2 using composite mode.</value>
     <comment>{StrBegin="NETSDK1163: "}</comment>
   </data>
-</root>
+  <data name="TrimmingWindowsFormsRequiresCom" xml:space="preserve">
+    <value>NETSDK1164: Windows Forms is not supported with trimming enabled. Please go to https://aka.ms/AAby5y8 for more details. To enable trimming and proceed, set `&lt;enableWindowsFormsTrimming&gt;true&lt;/enableWindowsFormsTrimming&gt;`. This configuration is not supported or recommended.</value>
```

Comment:
Is that link going to keep pointing to https://docs.microsoft.com/en-us/dotnet/core/deploying/trim-self-contained? If so, we already have a vanity link you can use here: https://aka.ms/dotnet-illink. If you plan on adding a new doc for this, it might be worth creating a separate vanity link like we did for the per-assembly warning: https://aka.ms/dotnet-illink/libraries (which will be updated once the new docs are available).